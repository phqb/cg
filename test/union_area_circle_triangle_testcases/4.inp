0 0 1
-0.24 1.36
-1.11 -0.77
1.71 -0.63