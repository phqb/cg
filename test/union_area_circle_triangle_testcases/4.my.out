2.4336231323